Remotion/Data/DomainObjects/DomainObject.cs

```diff
@@ -208,6 +208,7 @@ protected DomainObject ()
       RaiseReferenceInitializatingEvent();
 
       _needsLoadModeDataContainerOnly = true;
+      _strategy = new DomainObjectTransactionContextImplementation(this);
```

Comment:
Since it isn't set in the constructor, the field kicks out errors that it isn't initialized through the constructor even though it is not nullable. Am I missing something?

Reply:
Never mind, Mike's comments further down explain the situation.